src/torch_admp/pme.py

```
@@ -95,6 +95,12 @@ def __init__(
         """
         BaseForceModule.__init__(self, units_dict)
 
+        if rcut <= 0.0:
+            raise ValueError(f"rcut must be positive, got {rcut}")
+
+        if slab_axis not in np.arange(3):
+            raise ValueError(f"slab_axis must be 0/1/2, got {rcut}")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)